### DISCUSSION TITLE
How to display video on my vercel website?

### DISCUSSION THREAD

**Discussion by bkpleb** [vercel/community#154] [Help] [answered]
Okay so I want to display this video (https://www.youtube.com/watch?v=gPJWwiKnYGs&t=1s) on my vercel blog but I can only display the link. Is there a way to display the video running? 

Thanks!

**Comment by samcx** [comment, marked as answer]
You'll need to add an iframe → https://www.w3schools.com/html/html_youtube.asp. Let us know if you have any further questions or concerns.
